docs/root/development/performance/cpu_battery_impact.rst

```diff
@@ -0,0 +1,150 @@
+.. _dev_performance_cpu_battery:
+
+Analysis of CPU/battery impact
+==============================
+
+Modified versions of the "hello world" example apps were used to run these experiments:
+
+- `Android control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/control>`_
```

Comment:
I wonder if there is a way to protect these branches from being deleted. I know I sometimes go and prune old branches, so I would like something to yell at me if I try to delete these.

Reply:
GitHub keeps SHAs around, so I can pin to those instead (that way we could delete the branch and these links would still work)